packages/core/src/Link.js

```diff
@@ -13,6 +14,12 @@ const Link = styled.a`
   ${applyVariations('Link')}
 `
 
+const Link = props => {
+  const rel = props.target === '_blank' ? 'noopener' : null
```

Comment:
nitpick: destruct the props to `{ target }`.
Not so nitpick: pass remaining props to the component like `<StyledLink rel={rel} {...props} />` otherwise all props will be dropped and links probably won't work.

Reply:
Oof, my bad, definitely should have noticed that with the updated snapshots. Fixed and fixed!